src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1121,4 +1121,11 @@ protected T computeNext() {
     public static String getDateTimeForDisplay(final ZonedDateTime dateTime) {
         return dateTime.format(longDateTimeFormatter);
     }
+
+    /**
+     * A hack to ensure that numbers are always formatted in the US style.
```

Comment:
This is not really a hack :)  Method description should just be "Set the Locale to US English so that numbers will always be formatted in the US style."